examples/features/cancellation/README.md

```diff
@@ -0,0 +1,11 @@
+# Cancellation
+
+This example shows how clients can cancel in-flight RPCs.
```

Comment:
Add something like:

Client cancels the RPC by cancelling the RPC context. Both client and server will receive status with code `Cancel`.

Reply:
PTAL?